CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+<a name="6.0.0-beta.3"></a>
+# [6.0.0-beta.3](https://github.com/angular/material2/compare/6.0.0-beta-2...6.0.0-beta.3) (2018-02-27)
+
+
+### Bug Fixes
+
+* **cdk-dialog:** decorate dialog as an injectable ([#10052](https://github.com/angular/material2/issues/10052)) ([27e33e0](https://github.com/angular/material2/commit/27e33e0))
```

Comment:
some of these are repeated from the patch release below (artifact of the process, just gotta manually remove them)

Reply:
Done